zerver/actions/users.py

```diff
@@ -516,7 +516,15 @@ def get_owned_bot_dicts(
             "default_events_register_stream": botdict["default_events_register_stream__name"],
             "default_all_public_streams": botdict["default_all_public_streams"],
             "owner_id": botdict["bot_owner_id"],
-            "avatar_url": avatar_url_from_dict(botdict),
```

Comment:
If removing this line removes 100% coverage on `zerver/lib/avatar.py`, then the function should be removed in this commit itself, since we want each commit to pass the tests.